content/blog/2016/2016-04-07-pipeline-for-runs-on-hardware.adoc

```diff
@@ -0,0 +1,258 @@
+---
+layout: post
+title: Automating test runs on hardware with Pipeline-as-Code. Part 1
+tags:
+- jenkins
+- jenkins2
+- pipeline
+- embedded
+author: oleg_nenashev
+---
+
+In addition to Jenkins development, during last 8 years I've been involved into continuous integration for hardware and embedded projects.
+At JUC2015/London 
+link:https://www.cloudbees.com/jenkins/juc-2015/presentations/JUC-2015-Europe-Jenkins-Based-CI-for-Nenashev.pdf[I have conducted a talk] about common automation challenges in the area.
+
+In this blog post I would like to concentrate on link:https://jenkins.io/doc/pipeline/[Pipeline] (fka Workflow), which is a new automation-as-code ecosystem for Jenkins.
```

Comment:
I generally always use relative URLs for content posted on the site, that safe-guards us against future domain changes :smiling_imp: 

> (fka Workflow)

"fka" is not an acronym I've seen used, "formerly known as Workflow" is preferable.

> which is a new automation-as-code ecosystem for Jenkins

This statement doesn't parse very well, and is contrary to verbiage we've used already. "Automation as code" is not a phrase we use anywhere else. I suggest rephrasing to something like "I would like to concentrate on Pipeline (formerly known as Workflow), which is a new tool in Jenkins that enables implementing a delivery pipeline in code." (or something like tha